### DISCUSSION TITLE
Add a delay to Side bar extend on hover

### DISCUSSION THREAD

**Discussion by RABINGG** [zen-browser/desktop#2416] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Add a bit of delay to the expand on hover like in edge. This makes just for switching tabs not need the expanded sidebar

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.6 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1811/add-a-delay-to-side-bar-extend-on-hover">ZEN-1811 Add a delay to Side bar extend on hover</a></p>

**Reply by dglinder** [reply]
Any word on this?  I really like the side bar for the vertical tabs, but when I go into GMail and clik on the "Mail" button on the left to get to the "Compose" or the Labels/Folders, most of the time I overshoot and the Firefox Sidebar pops up and covers the GMail menus.

I've had to move the side bar to the right side of the Firefox menu, but it's not the "natural" location for me - I've used other side bar add-ons and always had them on the left.

Looking through the `about:config` features for the `sidebar.*` settings doesn't have an appropriate setting.

* The `sidebar.animation.duration-ms` doesn't help - it just makes it slower/faster to cover the GMail menus.
* The `sidebar.expandOnHover` doesn't apply either.
* The `sidebar.visibility` doesn't help either, unless there's an un-documented option other than `expand-on-hover` vs `always-show`.

**Comment by eMPee584** [comment]
I have another fix suggestion: add an **option to only show the sidebar when one of the modifiers `SHIFT` or `CTRL` is held** to prevent any unintentional popping up of the sidebar. Because on the right side, it happens when using the scroll bar, on the left when clicking on site menus..

**Reply by dglinder** [reply]
> I have another fix suggestion: add an **option to only show the sidebar when one of the modifiers `SHIFT` or `CTRL` is held** to prevent any unintentional popping up of the sidebar. Because on the right side, it happens when using the scroll bar, on the left when clicking on site menus..

I like this option - the `ctrl`/`shift` are quick options, and it could be implemented to make them enable OR DISABLE the feature on mouse over.

I.e. the default when turned on is to use the ctrl+mouseover to the hotzone to make it appear.  A flag could be set to invert that so it will stay hidden when ctrl+mouseover is used in that area.

This extra flag makes it easy to customize it based on what your use case dictates.'

But regardless, the `ctrl`/`shift` feature is a good idea.

**Comment by sguergachi** [comment]
is there a config for this in about:config?

**Reply by eMPee584** [reply]
not yet..
